pkg/apis/componentconfig/types.go

```diff
@@ -796,6 +796,10 @@ type KubeControllerManagerConfiguration struct {
 	EnableTaintManager bool
 	// If set to true NodeController will use taints to evict Pods from notReady and unreachable Nodes.
 	UseTaintBasedEvictions bool
+	// horizontalPodAutoscalerUseRESTClients causes the HPA controller to use REST clients
```

Comment:
i actually find this description clearer than what you use to document the flag in options.go